examples/react-apollo-crm/nhost/config.yaml

```diff
@@ -1,13 +1,5 @@
 metadata_directory: metadata
 services:
-  hasura:
-    image: hasura/graphql-engine:v2.10.1
-    environment:
-      hasura_graphql_enable_remote_schema_permissions: false
-  auth:
-    image: nhost/hasura-auth:0.13.2
-  storage:
-    image: nhost/hasura-storage:0.2.4
```

Comment:
images are pinned by [@nhost/sync-versions](https://github.com/nhost/nhost/tree/main/packages/sync-versions) to be sure we run any e2e test in the monorepo with the versions determined in [nhost-cloud.yaml](https://github.com/nhost/nhost/blob/main/nhost-cloud.yaml), and not the default versions in the cli that is used in our CI pipelines. Didn't husky run the sync with a pre-commit hook>